README
NAME
    Task::Error::Pure - Install the Error::Pure modules.

SYNOPSIS
     cpanm Task::Error::Pure

SEE ALSO
    Error::Pure
        Perl module for structured errors.

    Error::Pure::ANSIColor
        Error::Pure modules with Term::ANSIColor support.

    Error::Pure::ANSIColor::AllError
        Error::Pure module with full backtrace.

    Error::Pure::ANSIColor::Die
        Error::Pure module with classic die.

    Error::Pure::ANSIColor::Error
        Error::Pure module with error on one line with informations.

    Error::Pure::ANSIColor::ErrorList
        Error::Pure module with list of errors in one line with
        informations.

    Error::Pure::ANSIColor::Print
        Error::Pure module for simple error print.

    Error::Pure::ANSIColor::PrintVar
        Error::Pure module for simple error print with all variables with
        ANSIColor support.

    Error::Pure::AllError
        Error::Pure module with full backtrace.

    Error::Pure::Always
        Perl module for rewrite die by err from Error::Pure module.

    Error::Pure::Die
        Error::Pure module with classic die.

    Error::Pure::Error
        Error::Pure module with error on one line with informations.

    Error::Pure::ErrorList
        Error::Pure module with list of errors in one line with
        informations.

    Error::Pure::HTTP::AllError
        Error::Pure module with full backtrace over HTTP.

    Error::Pure::HTTP::Error
        Error::Pure module with error on one line over HTTP.

    Error::Pure::HTTP::ErrorList
        Error::Pure module with list of errors in one line with informations
        over HTTP.

    Error::Pure::HTTP::JSON
        Error::Pure module for JSON output over HTTP.

    Error::Pure::HTTP::JSON::Advance
        Error::Pure module for JSON output with additional parameters over
        HTTP.

    Error::Pure::HTTP::Print
        Error::Pure module for simple error print over HTTP.

    Error::Pure::JSON
        Error::Pure module for JSON output.

    Error::Pure::JSON::Advance
        Error::Pure module for JSON output with additional parameters.

    Error::Pure::NoDie
        Error::Pure module for simple print instead die.

    Error::Pure::Output::ANSIColor
        ANSIColor Output subroutines for Error::Pure.

    Error::Pure::Output::JSON
        Output JSON subroutines for Error::Pure.

    Error::Pure::Output::Text
        Output subroutines for Error::Pure.

    Error::Pure::Output::Tags::HTMLCustomPage
        Error::Pure HTML output helper.

    Error::Pure::Print
        Error::Pure module for simple error print.

    Error::Pure::PrintVar
        Error::Pure module for simple error print with all variables.

REPOSITORY
    <https://github.com/michal-josef-spacek/Task-Error-Pure>

AUTHOR
    Michal Špaček <mailto:skim@cpan.org>

    <http://skim.cz>

LICENSE AND COPYRIGHT
    © 2015-2023 Michal Josef Špaček

    Artistic License

    BSD 2-Clause License

VERSION
    0.05



Changes
0.05

0.04 2023-11-22T16:55:02+01:00
 - Add Error::Pure::ANSIColor dist.
 - Add Error::Pure::Output::ANSIColor dist.
 - Fix Makefile.PL fails with no '.' in @INC.
 - Improve LICENSE AND COPYRIGHT section in doc.
 - Move bugtracker to github.
 - Update Error::Pure to 0.31 version.
 - Update Error::Pure::Always to 0.07 version.
 - Update Error::Pure::HTTP to 0.15 version.
 - Update Error::Pure::JSON to 0.08 version.
 - Update Error::Pure::NoDie to 0.05 version.
 - Update Error::Pure::Output::JSON to 0.11 version.
 - Update Error::Pure::Output::Text to 0.24 version.
 - Update Module::Install to 1.21 version.
 - Update author github username.
 - Update author name.
 - Update copyright years.

0.03 2015-10-30T08:49:55+01:00
 - Add Error::Pure::HTTP::JSON::Advance to SEE ALSO section in doc.
 - Add Error::Pure::JSON::Advance to SEE ALSO section in doc.
 - Add Error::Pure::Output::Tags::HTMLCustomPage 0.03 version.
 - Update Error::Pure::HTTP to 0.14 version.
 - Update Error::Pure::HTTP::JSON to 0.05 version.
 - Update Error::Pure::JSON to 0.07 version.

0.02 2015-09-16T21:15:57+02:00
 - Fix REPOSITORY section in doc.
 - Improve and fix SEE ALSO section in doc.
 - Move admin tests to xt/ directory.
 - Update Module::Install to 1.16 version.
 - Update Error::Pure to 0.24 version.
 - Update Error::Pure::Always to 0.06 version.
 - Update Error::Pure::NoDie to 0.04 version.
 - Update Error::Pure::Output::JSON to 0.10 version.
 - Update Error::Pure::Output::Text to 0.22 version.

0.01 2015-01-14T19:05:15+01:00
 - First version


Pure.pm
package Task::Error::Pure;

use strict;
use warnings;

our $VERSION = 0.05;

1;

__END__

=pod

=encoding utf8

=head1 NAME

Task::Error::Pure - Install the Error::Pure modules.

=head1 SYNOPSIS

 cpanm Task::Error::Pure

=head1 SEE ALSO

=over

=item L<Error::Pure>

Perl module for structured errors.

=item L<Error::Pure::ANSIColor>

Error::Pure modules with Term::ANSIColor support.

=item L<Error::Pure::ANSIColor::AllError>

Error::Pure module with full backtrace.

=item L<Error::Pure::ANSIColor::Die>

Error::Pure module with classic die.

=item L<Error::Pure::ANSIColor::Error>

Error::Pure module with error on one line with informations.

=item L<Error::Pure::ANSIColor::ErrorList>

Error::Pure module with list of errors in one line with informations.

=item L<Error::Pure::ANSIColor::Print>

Error::Pure module for simple error print.

=item L<Error::Pure::ANSIColor::PrintVar>

Error::Pure module for simple error print with all variables with ANSIColor support.

=item L<Error::Pure::AllError>

Error::Pure module with full backtrace.

=item L<Error::Pure::Always>

Perl module for rewrite die by err from Error::Pure module.

=item L<Error::Pure::Die>

Error::Pure module with classic die.

=item L<Error::Pure::Error>

Error::Pure module with error on one line with informations.

=item L<Error::Pure::ErrorList>

Error::Pure module with list of errors in one line with informations.

=item L<Error::Pure::HTTP::AllError>

Error::Pure module with full backtrace over HTTP.

=item L<Error::Pure::HTTP::Error>

Error::Pure module with error on one line over HTTP.

=item L<Error::Pure::HTTP::ErrorList>

Error::Pure module with list of errors in one line with informations over HTTP.

=item L<Error::Pure::HTTP::JSON>

Error::Pure module for JSON output over HTTP.

=item L<Error::Pure::HTTP::JSON::Advance>

Error::Pure module for JSON output with additional parameters over HTTP.

=item L<Error::Pure::HTTP::Print>

Error::Pure module for simple error print over HTTP.

=item L<Error::Pure::JSON>

Error::Pure module for JSON output.

=item L<Error::Pure::JSON::Advance>

Error::Pure module for JSON output with additional parameters.

=item L<Error::Pure::NoDie>

Error::Pure module for simple print instead die.

=item L<Error::Pure::Output::ANSIColor>

ANSIColor Output subroutines for Error::Pure.

=item L<Error::Pure::Output::JSON>

Output JSON subroutines for Error::Pure.

=item L<Error::Pure::Output::Text>

Output subroutines for Error::Pure.

=item L<Error::Pure::Output::Tags::HTMLCustomPage>

Error::Pure HTML output helper.

=item L<Error::Pure::Print>

Error::Pure module for simple error print.

=item L<Error::Pure::PrintVar>

Error::Pure module for simple error print with all variables.

=back

=head1 REPOSITORY

L<https://github.com/michal-josef-spacek/Task-Error-Pure>

=head1 AUTHOR

Michal Špaček L<mailto:skim@cpan.org>

L<http://skim.cz>

=head1 LICENSE AND COPYRIGHT

© 2015-2023 Michal Josef Špaček

Artistic License

BSD 2-Clause License

=head1 VERSION

0.05

=cut


Makefile.PL
use lib '.';
use strict;
use warnings;

use inc::Module::Install;

# Definition.
abstract 'Install the Error::Pure modules.';
author 'Michal Josef Spacek <skim@cpan.org>';
author_requires 'English' => 0;
author_requires 'File::Object' => 0.08;
author_requires 'Test::Pod' => 0;
author_requires 'Test::Pod::Coverage' => 0;
license 'bsd';
name 'Task-Error-Pure';
readme_from 'Pure.pm';
recursive_author_tests('xt');
requires 'Error::Pure' => 0.31;
requires 'Error::Pure::ANSIColor' => 0.29;
requires 'Error::Pure::Always' => 0.07;
requires 'Error::Pure::HTTP' => 0.15;
requires 'Error::Pure::HTTP::JSON' => 0.05;
requires 'Error::Pure::JSON' => 0.08;
requires 'Error::Pure::NoDie' => 0.05;
requires 'Error::Pure::Output::ANSIColor' => 0.04;
requires 'Error::Pure::Output::JSON' => 0.11;
requires 'Error::Pure::Output::Tags::HTMLCustomPage' => 0.03;
requires 'Error::Pure::Output::Text' => 0.24;
requires 'perl' => '5.6.0';
resources 'bugtracker' => 'https://github.com/michal-josef-spacek/Task-Error-Pure/issues';
resources 'homepage' => 'https://github.com/michal-josef-spacek/Task-Error-Pure';
resources 'repository' => 'git://github.com/michal-josef-spacek/Task-Error-Pure';
test_requires 'Test::More' => 0;
test_requires 'Test::NoWarnings' => 0;
tests_recursive;
version '0.05';

# Run.
WriteAll();
